src/Tickets/Site_Health/Subsections/Features/Tickets_Commerce_Subsection.php

```diff
@@ -0,0 +1,262 @@
+				);
+				if ( isset( $matches[0] ) ) {
+					// Convert to a standard number format (replace comma with period).
+					$number = floatval(
+						str_replace(
+							',',
+							'.',
+							$matches[0]
+						)
+					);
+					$total  += $number;
+
+					++$count;
+				}
+			}
+		}
+
+		// Calculate the average price, avoid division by zero.
+		$tickets_commerce_average_price = $count > 0 ? $total / $count : 0;
+
+		// Format the average price with two decimal points.
+		$tickets_commerce_formatted_average_price = number_format(
+			$tickets_commerce_average_price,
+			2,
+			'.',
+			','
+		);
+
+		return $tickets_commerce_formatted_average_price;
+	}
+
+	/**
+	 * Determines if Tickets Commerce is in test mode.
+	 *
+	 * @return string 'True' if Tickets Commerce is in test mode, 'False' otherwise.
+	 */
+	private function is_tickets_commerce_test_mode(): string {
+		return $this->get_boolean_string(
+			tribe_get_option(
+				'tickets-commerce-test-mode',
+				false
+			)
+		);
+	}
+
+	/**
+	 * Checks if Stripe is connected with Tickets Commerce.
+	 *
+	 * @return string 'True' if Stripe is connected, 'False' otherwise.
+	 */
+	private function is_tickets_commerce_stripe_connected(): string {
+		return tribe( Stripe_Gateway::class )->is_enabled() ? 'True' : 'False';
+	}
+
+	/**
+	 * Determines if PayPal is connected with Tickets Commerce.
+	 *
+	 * @return string 'True' if PayPal is connected, 'False' otherwise.
+	 */
+	private function is_tickets_commerce_paypal_connected(): string {
+		return tribe( PayPal_Gateway::class )->is_enabled() ? 'True' : 'False';
+	}
+
+	/**
+	 * Retrieves the currency code set in Tickets Commerce.
+	 *
+	 * @return string The currency code.
+	 */
+	private function get_tickets_commerce_currency_code(): string {
+		return tribe_get_option(
+			'tickets-commerce-currency-code'
+		);
+	}
+
+	/**
+	 * Gets the currency position setting from Tickets Commerce.
+	 *
+	 * @return string The currency position.
+	 */
+	private function get_tickets_commerce_currency_position(): string {
+		return tribe_get_option(
+			'tickets-commerce-currency-position'
+		);
+	}
+
+
+	/**
+	 * Obtains the decimal separator setting from Tickets Commerce.
+	 *
+	 * @return string The decimal separator.
+	 */
+	private function get_tickets_commerce_decimal_separator(): string {
+		return tribe_get_option(
+			'tickets-commerce-currency-decimal-separator'
+		);
+	}
+
+	/**
+	 * Retrieves the thousands separator setting from Tickets Commerce.
+	 *
+	 * @return string The thousands separator.
+	 */
+	private function get_tickets_commerce_thousands_separator(): string {
+		return tribe_get_option(
+			'tickets-commerce-currency-thousands-separator'
+		);
+	}
+
+	/**
+	 * Gets the number of decimals setting from Tickets Commerce.
+	 *
+	 * @return string The number of decimals.
+	 */
+	private function get_tickets_commerce_number_of_decimals(): string {
+		return tribe_get_option(
+			'tickets-commerce-currency-number-of-decimals'
+		);
```

Comment:
Default is `2` (`TEC/Tickets/Commerce/Utils/Currency::$currency_code_number_of_decimals`):
```suggestion
		return tribe_get_option(
			'tickets-commerce-currency-number-of-decimals',
			'2'
		);
```